content/rs/administering/access-control/password-rotation.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: Database Password Rotation
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+To maintain secure access to your Redis Enterprise Software (RS) cluster and databases,
+most organizational security policies require that you rotate user passwords on a regular basis.
+
+For user access to the RS web UI,
+you can set a [password expiration policy]({{< relref "/rs/administering/access-control/#setting-local-user-password-expiration" >}}) that prompts the user to change their password.
+However, for database connections that rely on password authentication,
+you need to allow for authentication with the existing password while you rollout the new password to your systems.
+
+With the RS REST API, you can add additional passwords to an RS user account for authentication to the database or the Admin Console and API.
+After the old password is replaced in the database connections,
+just delete the old password to finish the password rotation process.
+
+{{ warning }}
+If you reset the password for a user in the RS Admin Console,
```

Comment:
I had to read this twice, so I think we should be more explicit:

"Multiple password are supported only using the REST API. If you reset a user password using the Admin Console, then the new password will replace all existing passwords for that user.